src/main/java/picard/sam/PositionBasedDownsampleSam.java

```diff
@@ -55,41 +45,112 @@
 
 
 /**
- * Class to downsample a BAM file while respecting that we should either get rid
- * of both ends of a pair or neither end of the pair. In addition, this program uses the read-name
- * and extracts the position within the tile whence the read came from. The downsampling is based on this position.
- * <p/>
- * Note 1: This is technology and read-name dependent. If your read-names do not have coordinate information, or if your
- * BAM contains reads from multiple technologies (flowcell versions, sequencing machines) this will not work properly.
- * This has been designed with Illumina MiSeq/HiSeq in mind.
- * <p/>
- * Note 2: The downsampling is _not_ random. It is deterministically dependent on the position of the read within its tile. Specifically,
- * it draws out an ellipse that covers a FRACTION fraction of the area and each of the edges and uses this to determine whether to keep the
- * record. Since reads with the same name have the same position (mates, secondary and supplemental alignments), the decision will be the same for all of them.
- * <p/>
- * Finally, the code has been designed to simulate sequencing less as accurately as possible, not for getting an exact downsample fraction.
- * In particular, since the reads may be distributed non-evenly within the lanes/tiles, the resulting downsampling percentage will not be accurately
- * determined by the input argument FRACTION. One should re-MarkDuplicates after downsampling in order to "expose" the duplicates whose representative has
+ * <h3>Summary</h3>
+ * Class to downsample a SAM/BAM file based on the position of the read in a flowcell. As with {@link DownsampleSam}, all the
+ * reads with the same queryname are either kept or dropped as a unit.
+ * <br/>
+ *
+ * <h3>Details</h3>
+ * The downsampling is <b>not</b> random (and there is no random seed). It is deterministically determined by the position
+ * of each read within its "tile". Specifically, it draws an ellipse that covers a {@link #FRACTION} of the total tile's
+ * area and of all the edges of the tile. It uses this area to determine whether to keep or drop the record. Since reads
+ * with the same name have the same position (mates, secondary and supplemental alignments), the decision will be the
+ * same for all of them. The main concern of this downsampling method is that due to "optical duplicates" downsampling
+ * randomly can create a result that has a different optical duplciate rate, and therefore a different estimated library
```

Comment:
randomly can is the correct order (with a different meaning) if you have a better wording I'm open to it. 

"downsampling randomly" (as opposed to "position-based downsampling) can ....

Reply:
Ah, got it!

Follow-up: I'm not sure what the sentence means by "The main concern of this downsampling method". Is it a concern because it's a downside of the chosen method which can cause discrepancies when compared to downsampling randomly? Or is it a concern about downsampling randomly that motivated using an alternate method?